### ISSUE TITLE
Checkbox and Radio button keyboard focus indicator not visible.

### ISSUE THREAD

**Issue by akawanishi** [jpmorganchase/salt-ds#4416]
### Latest version

- [X] I have tested the latest version

### Description

Checkbox does not have a focus indicator when item is focused with keyboard.
Radio button does have a change in color but not visible. I believe it is missing the dotted line as other components do for consistency. It is also hard to tell that it is focused when the radio button is checked and focused as it is same color.

### Steps to reproduce

keyboard focus on Checkbox
keyboard focus on Radio button

### Expected behavior

should have a dashed dotted line around the component just like other Salt compoenents

### Package name(s)

Core (@salt-ds/core)

### Package version(s)

latest

### Browser

- [X] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

**Comment by joshwooding** [comment]
Duplicate of #4342. This should be fixed in https://github.com/jpmorganchase/salt-ds/releases/tag/@salt-ds/core@1.37.2
